### ISSUE TITLE
Ensure `?mode=vr` forces stereo

### ISSUE THREAD

**Issue by cvan** [aframevr/aframe-core#704]
https://aframe.io/aframe/examples/showcase-shopping/?mode=vr doesn't seem to force stereo. Perhaps we want `?mode=stereo`, idk.

Per https://webvr.slack.com/archives/declarative-vr/p1450387507000072

**Comment by caseyyee** [comment]
Yeah, I think its useful to force stereo.   You wouldn't be able to so otherwise unless you are on mobile or a VR enabled browser.

**Comment by caseyyee** [comment]
Also would be useful to have a function call to do the same.

**Comment by cvan** [comment]
And a `postMessage` call.

**Comment by evilgouky** [comment]
Hi all,

I'm looking for a solution to be able to force the vr view on smartphone when my webpage is launched.
Anyone can help me with that ?

Thank you in advance

**Comment by cvan** [comment]
I'm working on this now. I'll update the bug when it's ready.
